graphql/server.go

```diff
@@ -157,9 +159,12 @@ func (c *conn) handleSubscribe(id string, subscribe *subscribeMessage) error {
 		ctx = c.makeCtx(ctx)
 
 		start := time.Now()
+		span, ctx := opentracing.StartSpanFromContext(ctx, "thunder.subscription")
+		ext.SamplingPriority.Set(span, 1)
```

Comment:
This can be set to 0 to disable the trace. This is effectively a no-op for now, but we can follow up to set this for a "debug" flag. Considering either adding a `trace` flag on the `InMessage` struct, or adding an incoming http header for the entire connection.